daemon/list.go

```diff
@@ -516,6 +517,21 @@ func (daemon *Daemon) filterVolumes(vols []volume.Volume, filter filters.Args) (
 				continue
 			}
 		}
+		if filter.Include("label") {
+			v, err := daemon.volumes.Get(vol.Name())
```

Comment:
the volume should already have the labels populated, I think, there should not be need to call `Get` again.